runexporterapp.sh

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/bash
 
-cd /ironic-prometheus-exporter
+cd /usr/lib/python2.7/site-packages/ironic_prometheus_exporter/
```

Comment:
This is not needed any more, uwsgi should be able to find the package in its standard location

Reply:
++ We can't expect a specific python version or path.

Reply:
Ok, I'm trying to figure out how to not use the path =)

Reply:
Updated =D